cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/grudra7714/divvy-master/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/home/grudra7714/divvy-master/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140462682848656: c test.c -> test.c.1.o}\n[\'/usr/bin/gcc\', \'-DHAVE_CEIL=1\', \'../test.c\', \'-c\', \'-o\', \'test.c.1.o\']\n'
